vue/src/components/loki/LogLabelValuesCont.vue

```diff
@@ -52,23 +68,44 @@ export default defineComponent({
       }
     })
 
-    const internalLabels = computed((): Label[] => {
-      return labelValues.items.map((value: string): Label => {
-        return { name: value, selected: false }
+    const internalLabels = computed((): LabelItem[] => {
+      return labelValues.items.map((value: string): LabelItem => {
+        return { name: value, selected: false, label: props.label || '' }
       })
     })
 
     const labels = computed((): Label[] => {
```

Comment:
Looks like `LabelItem` and `Label` interfaces are the same? Let's only keep `Label`?